client/src/components/pages/home/PastTourneyCard.js

```diff
@@ -0,0 +1,52 @@
+import { Image, Typography } from "antd";
+import Text from "antd/lib/typography/Text";
+import React, { useState, useEffect } from "react";
+
+import ContentManager from "../../../ContentManager";
+
+import "./PastTourneyCard.css";
+import { Link } from "@reach/router";
+
+/**
+ * @typedef tourney
+ * @property {string} displayName
+ * @property {string} code
+ * @property {number} year
+ * @property {string} codeAndDivision
+ * @property {string} fullCode
+ */
+/**
+ * @param {object} props
+ * @param {tourney} props.tourney
+ */
+export default function PastTourneyCard({ tourney }) {
+  const [tourneyContent, SetTourneyContent] = useState(null);
```

Comment:
nit: `setTourneyContent`

Reply:
Fixed in 400d747fcda341fb788f49c155bcf464530e1af2